src/sweeper/backup.py

```diff
@@ -4,3 +4,19 @@
 backup.py
 A module that creates a gdb if it doesn't exist and inserts a feature class
 '''
+import arcpy 
+
+def backup_data(source_data, out_path):
+    # if the gdb does not exist, then create it
+    if not arcpy.Exists(out_path):
+        out_dir = out_path.rsplit('\\', 1)[0]
+        gdb_name = out_path.rsplit('\\', 1)[1]
+        arcpy.CreateFileGDB_management(out_dir, gdb_name)
+    
+    # get source_data feature class name
+    fc_name = source_data.rsplit('\\',1)[1]
+
+    # backup the source feature class
+    arcpy.Copy_management(source_data, out_path + '\\' + fc_name)
```

Comment:
What happens if the table already exists?

Reply:
if the table exists, then that db will be used as the location for the backup.  is this a better code? 

`    #: check if the db exists
    if not arcpy.Exists(out_path):
        #: if db does not exist, then create it
        out_dir = out_path.rsplit('\\', 1)[0]
        gdb_name = out_path.rsplit('\\', 1)[1]
        arcpy.CreateFileGDB_management(out_dir, gdb_name)
    else:
        #: if db exists, use it as location to load backup file
        pass`

Reply:
oops, originally I misread your question.  yup, good question.  should we always append the datetime after the table name?  thoughts?  @steveoh @eneemann

Reply:
I think appending the datetime and/or "backup" into the name would work

Reply:
i added a datetime to the table name.  see what you think.  it's in my most recent commit.